### DISCUSSION TITLE
Axios ad Tar dependency libraries have vulnerabilities causing security issues

### DISCUSSION THREAD

**Discussion by Sujietha** [nrwl/nx#23309] [Questions] [answered]
Axios and Tar dependency libraries have vulnerabilities in 18.0.x version [latest] causing security issues. Please update axios and tar to latest versions

**Comment by jaysoo** [comment, marked as answer]
Both `tar` and `follow-redirects` (the `axios` vulnerability) are deps of deps, so we don't have control over their versions. I recommend you open issues against those projects instead. This isn't something we can handle on our end.

Run `npm why tar` and `npm why follow-redirects` (or the pnpm/yarn equivalent) to see what packages are using them.

For your own repo, you can use `overrides` or `resolutions` in your `package.json` file (depending on your package manager) to force the patched versions of affected packages.

**Reply by Sujietha** [reply]
thanks will try
